tools/manifest.json

```diff
@@ -57,5 +59,20 @@
 	},
 	"Clients": {
 		"owner": [9,9,9]
+	},
+	"Secrets": {
+		"web_cert_shared": {
+			"Shared": true,
+			"Size": 256,
+			"Type": "cert-ecdsa",
+			"Cert": {
+				"SerialNumber": 1337,
```

Comment:
I'm not sure, but I'd guess best practice would be to let the CA generate the SN. If that's true, we should stick to it in this demo.